x-pack/plugins/security/server/lib/authorization/privileges/privileges.test.ts

```diff
@@ -0,0 +1,631 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+// @ts-ignore
+import { Feature } from '../../../../xpack_main/types';
+import { Actions } from '../actions';
+import { privilegesFactory } from './privileges';
+
+const actions = new Actions('1.0.0-zeta1');
+
+describe('features', () => {
```

Comment:
Can we add tests to ensure that the privileges factory & builder can handle features that don't have privileges defined?